.semaphore/semaphore.yml

```
@@ -83,10 +83,22 @@ blocks:
             # Check shell scripts for issues.
             - "shellcheck package/*.sh package/deb-postinst"
 
-            # Run build and tests in Nix, and push the result to Cachix.
+            # Print working directory for debugging purposes
+            - "pwd"
+
+            # Run build and tests in Nix
             # Because this is a public repository, and not everyone is using Nix,
             # we keep the stack setup around. The Nix build however is used in production.
-            - "nix-build --no-out-link release.nix | cachix push channable-public"
+            - "nix-build --no-out-link release.nix >nix-store-location"
```

Optional suggestion: if it's still desired that the resulting store path is shown, I suggest using `| tee nix-store-location` to make the command itself print the output.

```suggestion
            - "nix-build --no-out-link release.nix | tee nix-store-location"
```

---

Oh nevermind this, this would of course still cause the problem with not detecting the failure because it'd pipe the output again.

---

The `tee` here wouldn't work, as we would fall back into the existing issue.  If `nix-build` were to fail, the CI would not detect it as it would take the error code of `tee`.